src/System.Buffers.Experimental/System/Buffers/ReadOnlyBytes.cs

```diff
@@ -295,8 +301,8 @@ public static ReadOnlyBytes Create(params byte[][] buffers)
 
         public struct Cursor
```

Comment:
Should it be readonly?

Reply:
Yeah, I made it readonly. Though, honestly I am trying to figure out how to use Position as the cursor. i.e. it can be easily done but it might be confusing that the same type would represent a position in the sequence (linked list node) and a position withing the sequence of bytes. I need to think about it.